src/main/kotlin/org/rust/lang/core/psi/ext/RsAttr.kt

```diff
@@ -20,3 +22,11 @@ val RsAttr.owner: RsDocAndAttributeOwner?
         // Throw exception so that any problems with this property aren't silent and are instead easily findable
         else -> error("Unsupported attribute type: $this")
     }
+
+val RsAttr.isBuiltInAttribute: Boolean
+    get() {
+        val name = metaItem.name ?: return false
+        if (name !in STD_ATTRIBUTES && name != "start") return false
```

Comment:
Can we put `start` in `STD_ATTRIBUTES` list?

Reply:
`start` currently exists in the unstable list, I think maybe it can?